src/main/java/com/feywild/feywild/data/CommonTagsProvider.java

```diff
@@ -45,15 +43,77 @@ public void setup() {
+
+        this.block(ModBlockTags.AUTUMN_LOGS).add(
+                ModTrees.autumnTree.getLogBlock(),
+                ModTrees.autumnTree.getStrippedLogBlock(),
+                ModTrees.autumnTree.getWoodBlock(),
+                ModTrees.autumnTree.getStrippedWoodBlock());
+        this.copyBlock(ModBlockTags.AUTUMN_LOGS, ModItemTags.AUTUMN_LOGS);
+
+        this.block(ModBlockTags.SPRING_LOGS).add(
+                ModTrees.springTree.getLogBlock(),
+                ModTrees.springTree.getStrippedLogBlock(),
+                ModTrees.springTree.getWoodBlock(),
+                ModTrees.springTree.getStrippedWoodBlock());
+        this.copyBlock(ModBlockTags.SPRING_LOGS, ModItemTags.SPRING_LOGS);
+
+        this.block(ModBlockTags.SUMMER_LOGS).add(
+                ModTrees.summerTree.getLogBlock(),
+                ModTrees.summerTree.getStrippedLogBlock(),
+                ModTrees.summerTree.getWoodBlock(),
+                ModTrees.summerTree.getStrippedWoodBlock());
+        this.copyBlock(ModBlockTags.SUMMER_LOGS, ModItemTags.SUMMER_LOGS);
+
+        this.block(ModBlockTags.WINTER_LOGS).add(
+                ModTrees.winterTree.getLogBlock(),
+                ModTrees.winterTree.getStrippedLogBlock(),
+                ModTrees.winterTree.getWoodBlock(),
+                ModTrees.winterTree.getStrippedWoodBlock());
+        this.copyBlock(ModBlockTags.WINTER_LOGS, ModItemTags.WINTER_LOGS);
+
+        this.block(BlockTags.PLANKS).add(
+                ModTrees.autumnTree.getPlankBlock(),
+                ModTrees.springTree.getPlankBlock(),
+                ModTrees.summerTree.getPlankBlock(),
+                ModTrees.winterTree.getPlankBlock());
+        this.copyBlock(BlockTags.PLANKS, ItemTags.PLANKS);
+
+        this.block(BlockTags.WOODEN_FENCES).add(
+                ModTrees.autumnTree.getPlankBlock().getStairBlock(),
+                ModTrees.springTree.getPlankBlock().getStairBlock(),
+                ModTrees.summerTree.getPlankBlock().getStairBlock(),
+                ModTrees.winterTree.getPlankBlock().getStairBlock());
+        this.copyBlock(BlockTags.WOODEN_FENCES, ItemTags.WOODEN_FENCES);
+
+        this.block(BlockTags.WOODEN_SLABS).add(
+                ModTrees.autumnTree.getPlankBlock().getSlabBlock(),
+                ModTrees.springTree.getPlankBlock().getSlabBlock(),
+                ModTrees.summerTree.getPlankBlock().getSlabBlock(),
+                ModTrees.winterTree.getPlankBlock().getSlabBlock());
+        this.copyBlock(BlockTags.WOODEN_SLABS, ItemTags.WOODEN_SLABS);
+
+        this.block(BlockTags.WOODEN_STAIRS).add(
+                ModTrees.autumnTree.getPlankBlock().getStairBlock(),
+                ModTrees.springTree.getPlankBlock().getStairBlock(),
+                ModTrees.summerTree.getPlankBlock().getStairBlock(),
+                ModTrees.winterTree.getPlankBlock().getStairBlock());
+        this.copyBlock(BlockTags.WOODEN_STAIRS, ItemTags.WOODEN_STAIRS);
```

Comment:
LibX will take care of this automatically.

Reply:
At least I think so. If it doesn't, please open a LibX issue.

Reply:
https://github.com/noeppi-noeppi/LibX/issues/35